### DISCUSSION TITLE
Separate Shoes & Gloves

### DISCUSSION THREAD

**Discussion by cosmiccoincidence** [RE-SS3D/SS3D#869] [Game Design]
Starting discussion again on this idea:

https://trello.com/c/qp1V9t4Q/16-clothing-improvements

> Shoes & gloves are now separated into individuals.
> 
> Shoes/gloves can be stacked with an identical shoe/glove to create a pair that can be more easily stored, equipped, etc. Individual shoes/gloves need to be handled/equipped 1 at a time.
> 
> This allows for wearing different shoes/gloves on your feet/hands at the same time.
> 
> If your limb gets removed on 1 side, that garment can now stay equipped to the severed limb while the other garment is stays equipped to the other limb attached to the body. With ss13 pairs, oftentimes losing 1 limb is enough to destroy/lose your pair of shoes/gloves.
> 
> We'll need to decide if all shoes/gloves are universal or if there will be lefts and rights.

Please vote with a 'thumbs up' or 'thumbs down' if you like the idea.

And feel free to make a comment to share any thoughts you have.

(If you don't have a github account, you can also discuss this topic in the #game-design channel on our discord while tagging this issue using "#869")

**Comment by iamteapot422** [comment]
I don't see in what situation you would need two different gloves or shoes

**Reply by cosmiccoincidence** [reply]
Could be the new meta to wear insulation glove on one hand and a boxing glove on the other... IDK. That part is more an extra ability that would be possible with this.

**Comment by yacabo** [comment]
> If your limb gets removed on 1 side, that garment can now stay equipped to the severed limb while the other garment is stays equipped to the other limb attached to the body. With ss13 pairs, oftentimes losing 1 limb is enough to destroy/lose your pair of shoes/gloves.

Why not just make it so that that doesn't happen? It'd be a lot easier to just make it so that your gloves/shoes don't fall off when you loose a limb. Sure it might not be realistic but this is a video game and it'll remove a lot of headaches.

**Reply by cosmiccoincidence** [reply]
Definitely a feasible solution. With them being pairs we could probably make them separate child objects in the same item prefab, which could allow us to disable one if we needed to.
